eZ/Bundle/EzPublishCoreBundle/EventListener/CacheViewResponseListener.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\EventListener;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\Core\MVC\Symfony\View\LocationValueView;
+use eZ\Publish\Core\MVC\Symfony\View\View;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Configures the Response cache properties.
+ */
+class CacheViewResponseListener implements EventSubscriberInterface
+{
+    /**
+     * True if view cache is enabled, false if it is not.
+     *
+     * @var bool
+     */
+    private $enableViewCache;
+
+    /**
+     * True if TTL cache is enabled, false if it is not.
+     * @var
+     */
+    private $enableTtlCache;
+
+    /**
+     * Default ttl for ttl cache.
+     *
+     * @var int
+     */
+    private $defaultTtl;
+
+    public function __construct($enableViewCache, $enableTtlCache, $defaultTtl)
+    {
+        $this->enableViewCache = $enableViewCache;
+        $this->enableTtlCache = $enableTtlCache;
+        $this->defaultTtl = $defaultTtl;
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return [KernelEvents::RESPONSE => 'configureCache'];
+    }
+
+    public function configureCache(FilterResponseEvent $event)
+    {
+        if (!($view = $event->getRequest()->attributes->get('view')) instanceof View) {
```

Comment:
Maybe we should also check a dedicated parameter in the view to ensure the controller (or any service before getting here) did not already manipulate the response. This would allow to fully customize the cache.
What about having a `CachedView` decorating the main `View`, with dedicated properties / methods?